### PR TITLE
increase test coverage of /pkg/runtime

### PR DESCRIPTION
**Pull request by mandarjog**
**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1300)

### CODE REVIEW

**Comment by mandarjog** [comment]
```
I0914 18:25:57.803] ok  	istio.io/mixer/pkg/runtime	1.681s	coverage: 94.5% of statements
```
From codecov tests

File: pkg/runtime/env_test.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+			wg.Done()
+		})
+
+		wg.Add(1)
+		env.ScheduleDaemon(func() {
+			wg.Done()
+		})
+
+		env.ScheduleWork(func() {
+			panic("bye!")
```
Comment (jmuk) [inline pkg/runtime/env_test.go]: maybe t.Fatal rather than panic?

File: pkg/runtime/env_test.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		})
+
+		env.ScheduleDaemon(func() {
+			panic("bye!")
```
Comment (jmuk) [inline pkg/runtime/env_test.go]: ditto

File: pkg/runtime/handlerTable.go [hunk 3]
```diff
@@ -148,23 +148,26 @@ func (t *handlerTable) initHandler(he *HandlerEntry) {
...
-			glog.Warningf("Failed to write %v to buffer: %v", t, err)
 		}
 	}
+
+	if b == nil {
+		glog.Warningf("Falling back to fmt.Fprintf()", v)
+		b = []byte(fmt.Sprintf("%+v", v))
+	}
+
+	if _, err := w.Write(b); err != nil {
```
Comment (jmuk) [inline pkg/runtime/handlerTable.go]: this should be `_, err = ` instead of `:=`?

File: pkg/runtime/env_test.go (hunk 1 above)
Comment (mandarjog) [reply pkg/runtime/env_test.go]: The purpose of the test is to see how `ScheduleWork` and `ScheduleDaemon` behave when a function they are executing panic..

File: pkg/runtime/handlerTable.go (hunk 3 above)
Comment (mandarjog) [reply pkg/runtime/handlerTable.go]: fixed.

**Pull request merged**